fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -152,17 +152,36 @@ def self.buildlog_path
     #  running system
     def self.xcode_path
       return "" unless self.is_mac?
-      `xcode-select -p`.delete("\n") + "/"
+
+      if self.xcode_server?
+        # Xcode server always creates a link here
+        return "/Library/Developer/XcodeServer/CurrentXcodeSymlink/Contents/Developer"
```

Comment:
To what version of Xcode does this link to then? Maybe we should add a `UI.verbose` here to log that we're doing this

Reply:
This links to whatever XcodeServer has determined it will use.